src/app/contact/contact.component.scss

```diff
@@ -209,6 +212,7 @@
   display: flex;
   justify-content: flex-end;
   align-items: flex-start;
+  z-index: 10;
```

Comment:
This change seems to be present on both of your PRs. Please avoid duplicating code across PRs.